packages/test/test-end-to-end-tests/src/test/container.spec.ts

```diff
@@ -48,7 +48,8 @@ describe("Container", () => {
     async function loadContainer(props?: Partial<ILoaderProps>) {
         const loader = new Loader({
             ...props,
-            logger: ChildLogger.create(getTestLogger(), undefined, {all: {testDriverType: driver.type}}),
+            logger: ChildLogger.create(
+                getTestLogger(), undefined, {all: {hostName: `end-to-end tests (${driver.type})`}}),
```

Comment:
i don't like this here, as only tests use this library as well. this should come from the mocha hook of the logger

Reply:
i think hostName should be the test name

Reply:
or maybe hostName is the test file, and hostScenario is the testname?

Reply:
Reverting the use of `hostName`.  Will follow-up in an issue about what other metadata goes where in these logs.